experimental/devices/inky/inky.go

```diff
@@ -100,13 +107,26 @@ type Dev struct {
 	color Color
 	// Modifiable color of border.
 	border Color
+
+	bounds         image.Rectangle
```

Comment:
Since these are not mutable, they should go in the block above of immutable members.

Reply:
Done!